airflow/providers/databricks/operators/databricks.py

```diff
@@ -397,6 +396,7 @@ class DatabricksRunNowOperator(BaseOperator):
 
     Currently the named parameters that ``DatabricksRunNowOperator`` supports are
         - ``job_id``
+        - ``job_name``
```

Comment:
Missing documentation about the `job_name` parameter:
```
:param job_name: the name of the existing Databricks job. It must exists only one job with the specified name. ``job_id`` and ``job_name`` are mutually exclusive.
        This field will be templated.
```

Reply:
Thank you for the review, agree on all the points 👍  fixed here and in other places mentioned